src/uct/base/uct_cm.h

```diff
@@ -74,7 +74,7 @@ typedef struct uct_cm_base_ep {
         struct {
             /* On the server side - callback to process an incoming connection
              * establishment acknowledgment from the client */
-            uct_cm_ep_server_connect_callback_t connect_cb;
+            uct_cm_ep_server_notify_callback_t  notify_cb;
```

Comment:
pls update doc for this field with "notify"

Reply:
acknowledgment --> notification ?

Reply:
yes